queries/unix_heartbeat.kusto

```diff
@@ -2,6 +2,14 @@
 // Display the VM's reported availability during the last 5 minutes.
 Heartbeat
 | extend affected_object = toupper(tostring(split(_ResourceId, "/") [-1]))
+| join kind= leftouter (MonitoringResources_CL 
```

Comment:
It's possible to extract this query out and have it separate instead of an inline query. That helps readability and makes it so that the queries can be tested individually if needed. See [vm backup](https://github.com/qbeyond/terraform-azurerm-alz-monitoring/blob/main/queries/vm_backup.kusto) for example